OptimizelySDK/Optimizely.cs

```diff
@@ -281,5 +281,31 @@ public string GetVariation(string experimentKey, string userId, UserAttributes u
             Variation variation = DecisionService.GetVariation(experiment, userId, userAttributes);
             return variation == null ? null : variation.Key;
         }
+
+        /// <summary>
+        /// Force a user into a variation for a given experiment.
+        /// </summary>
+        /// <param name="experimentKey">The experiment key</param>
+        /// <param name="userId">The user ID</param>
+        /// <param name="variationKey">The variation key specifies the variation which the user will be forced into.
+        /// If null, then clear the existing experiment-to-variation mapping.</param>
+        /// <returns>A boolean value that indicates if the set completed successfully.</returns>
+        public bool SetForcedVariation(string experimentKey, string userId, string variationKey)
+        {
+            return Config.SetForcedVariation(experimentKey, userId, variationKey);
+        }
+
+        /// <summary>
+        /// Gets the forced variation key for the given user and experiment.  
+        /// </summary>
+        /// <param name="experimentKey">The experiment key</param>
+        /// <param name="userId">The user ID</param>
+        /// <returns>null|string The variation key.</returns>
+        public string GetForcedVariation(string experimentKey, string userId)
```

Comment:
@msohailhussain looks like we overlooked this part. This `GetForcedVariation` method should return a `Variation` object instead of a string. The reason being that all of our static-typed SDKs return the `Variation` object. Here is an example: https://github.com/optimizely/java-sdk/blob/master/core-api/src/main/java/com/optimizely/ab/Optimizely.java#L650

The reference we gave you was in PHP which returns a string because it's a dynamic language.